src/Microsoft.Extensions.ML/ModelLoaders/FileModelLoader.cs

```diff
@@ -105,7 +105,7 @@ public override ITransformer GetModel()
             return _model;
         }
 
-        private FileStream WaitForFile(string fullPath, FileMode mode, FileAccess access, FileShare share)
+        internal virtual FileStream WaitForFile(string fullPath, FileMode mode, FileAccess access, FileShare share)
```

Comment:
Can you add a comment to this like the method below has?
`//internal virtual for testing purposes.`